molecule/app_deploy_standalone/converge.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Deploy application to WildFly
+  hosts: all
+  gather_facts: false
+  vars_files:
+    - ../group_vars/all/vars.yml
+  tasks:
+    - name: "Deploy application using wildfly_app_deploy role"
+      ansible.builtin.include_role:
+        name: wildfly_app_deploy
+      vars:
```

Comment:
create a `vars.yml` file and put all the vars there and provide it via `vars_files`